roles/check-instance/defaults/main.yml

```diff
@@ -0,0 +1,38 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+allow_install_on_vm: false
+
+ansible_user: ansible
+
+required_cpu_cores: 4
+required_cpu_arch: x86_64
+
+required_mem_mb: 15000
+
+supported_distributions_19c:
```

Comment:
I'm concerned this list gets very complicated quickly, and challenging to keep up to date, as more versions and OSes are added.  For example, SuSE Linux, el8 kernels, Oracle 12cR1, 11gR2, 20c etc?  Might this be one place we could fall back on Oracle's own install prerequisite checks?

Reply:
This is a pre-install check, for obvious reasons we cannot include here the Oracle checks and the initial request was to eliminate as much as possible an install-time fail, this is the OS check part. The OS/Kernel combination is compared to the list and anything newer is accepted so the list needs to be updated only once, anything newer will be automatically included.
I could not find anything for the other distributions so we can have a baseline and start from there, but I wanted something official

Reply:
Thanks for your work here.  My preference would be to avoid duplicating too much of the Oracle checks, but rather check for what we need to get the toolkit itself to run.  How about removing the kernel versions entirely, and just focusing on OS major releases, ideally without making distinctions between Oracle versions.  Perhaps as simpe as OL7 or RHEL7?  (And I hope OL8/RHEL8 too?)